vader.py

```diff
@@ -0,0 +1,63 @@
+from nltk.sentiment.vader import SentimentIntensityAnalyzer
+from nltk import tokenize
+import nltk
+import ssl
+
+try:
+    _create_unverified_https_context = ssl._create_unverified_context
+except AttributeError:
```

Comment:
Since this is an exception should at least print something until logging is implemented. 

Overall, this SSL section could use a comment or two. I wasn't sure what it was at first.